src/utils.ts

```diff
@@ -306,12 +306,15 @@ export async function getSpaceUri(
   }
 }
 
-export async function getEnsOwner(ens: string): Promise<string | null> {
+export async function getEnsOwner(
+  ens: string,
+  network = '1'
+): Promise<string | null> {
   const registryAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
```

Comment:
@ChaituVR is goerli the same registry address?

Reply:
Yep same one